src/cdk/stepper/stepper.ts

```diff
@@ -156,15 +153,18 @@ export class CdkStep implements OnChanges {
   selector: '[cdkStepper]',
   exportAs: 'cdkStepper',
 })
-export class CdkStepper implements OnDestroy {
+export class CdkStepper implements AfterViewInit, OnDestroy {
   /** Emits when the component is destroyed. */
   protected _destroyed = new Subject<void>();
 
+  /** Used for managing keyboard focus. */
+  private _keyManager: FocusKeyManager<FocusableOption & {blur: () => void}>;
+
   /** The list of step components that the stepper is holding. */
   @ContentChildren(CdkStep) _steps: QueryList<CdkStep>;
 
   /** The list of step headers of the steps in the stepper. */
-  _stepHeader: QueryList<ElementRef>;
+  _stepHeader: QueryList<FocusableOption & {blur: () => void}>;
```

Comment:
should we add the `blur` method to `FocusableOption`? It makes sense to me that an option that can be focused can also be blurred

Reply:
I don't think it's necessary since moving the focus to another option will blur it automatically.